tests/unit/keyspace.tcl

```diff
@@ -7,10 +7,10 @@ start_server {tags {"keyspace" "external-ok"}} {
     } {}
 
     test {Vararg DEL} {
-        r set foo1 a
-        r set foo2 b
-        r set foo3 c
-        list [r del foo1 foo2 foo3 foo4] [r mget foo1 foo2 foo3]
+        r set foo1{t} a
+        r set foo2{t} b
+        r set foo3{t} c
+        list [r del foo1{t} foo2{t} foo3{t} foo4{t}] [r mget foo1{t} foo2{t} foo3{t}]
```

Comment:
maybe this test, when it detects a non-cluster mode, should avoid using tags?
this way we can use the redis test suite in a non-cluster mode to run against a proxy, and see that it is able to handle that case.

Reply:
Maybe we just need an explicit multi key command test?

Reply:
Arguably, this is the one. 
But maybe we need two, one for multi key with tags, and one with no tags. 
So either we add another test, and add a skip flag for one. 
Or we keep just one test, and change its behaviour depending on the cluster variable..
I guess two tests is nicer (less hackish)